reconcile/terraform_resources.py

```diff
@@ -300,6 +301,17 @@
   namespaces: namespaces_v1 {
     name
     managedTerraformResources
+    terraformProviderResources {
+      provider
+      provisioner {
+        name
+      }
+      ... on NamespaceTerraformProviderResourceAWS_v1 {
+        resources {
+          %s
```

Comment:
As above, we should aim to unify these rendering styles.

Reply:
since there are multiple occurrences in this file already